lib/tasks/b2b.rake

```diff
@@ -0,0 +1,19 @@
+namespace :b2b do
+  desc 'Create corporate article type'
+  task create_type: :environment do
+    ActiveRecord::Base.logger = Logger.new(STDOUT)
```

Comment:
Actually I prefer to see the select/insert SQL statements in the output when I am running the task (even with the puts more humanized).